site/profile/manifests/core/icinga_master.pp

```diff
@@ -275,6 +275,38 @@
     api_password  => $api_pwd,
     require       => Mysql::Db[$mysql_director_db],
   }
+  ##Director Daemon
+  user { 'icingadirector':
+    ensure => 'present',
+    system => true,
+    shell  => '/bin/false',
+    groups => 'icingaweb2',
+    home   => '/var/lib/icingadirector',
+  }
+  ->file { '/var/lib/icingadirector':
+    ensure => 'directory',
+    mode   => '0750',
+    owner  => 'icingadirector',
+    group  => 'icingaweb2',
+  }
+  class { '::php::globals':
```

Comment:
I recall you stating this module wasn't necessary because php was managed by icingaweb2, what changed?

Reply:
the icinga-director daemon requires soap and process (posix)